website/docs/reference/commands/source.md

```diff
@@ -12,6 +12,64 @@ The `dbt source` command provides subcommands that are useful when working with
 
 If your dbt project is [configured with sources](/docs/build/sources), then the `dbt source freshness` command will query all of your defined source tables, determining the "freshness" of these tables. If the tables are stale (based on the `freshness` config specified for your sources) then dbt will report a warning or error accordingly. If a source <Term id="table" /> is in a stale state, then dbt will exit with a nonzero exit code.
 
+You can also use [source freshness](/docs/deploy/source-freshness) commands help make sure the data you get is new and not old or outdated. 
+
+### Configure source freshness
+
+To configure source freshness in dbt:
+
+- Add a [`freshness`](/reference/resource-properties/freshness) block to your source in the .yml file.
+- Specify `warn_after` and `error_after` thresholds.
+- Include [`loaded_at_field`](/reference/resource-properties/freshness#loaded_at_field) for each table.
+- Use the [`dbt source freshness`](/reference/commands/source#dbt-source-freshness) command to check freshness.
+
+<File name='models/<filename>.yml'>
+
+```yaml
+
+version: 2
+
+sources:
+  - name: jaffle_shop
+    database: raw
+
+    freshness:
+      warn_after: {count: 12, period: hour}
+      error_after: {count: 24, period: hour}
+
+    loaded_at_field: _etl_loaded_at
+
+    tables:
+      - name: customers
+
+      - name: orders
+        freshness:
+          warn_after: {count: 6, period: hour}
+          error_after: {count: 12, period: hour}
+          filter: datediff('day', _etl_loaded_at, current_timestamp) < 2
+
+      - name: product_skus
+        freshness: null
+
+```
+</File>
+
+This helps to monitor the data pipeline health.
+
+You can also configure source freshness in the execution settings within your job in dbt Cloud. For more information, refer to [Enabling source freshness snapshots](/docs/deploy/source-freshness#enabling-source-freshness-snapshots).
+
+### Source freshness commands
+
+Source freshness commands ensure you're receiving the most up-to-date, relevant, and accurate information. 
+
+Some of the typical commands you can use are:
+
+| **Command**                                                                 | **Description**                  | 
+| ----------------------------------------------------------------------------| ---------------------------------|
+|[`dbt source freshness`](/reference/commands/source#dbt-source-freshness)    |Checks the "freshness" for all sources.|
+|[`dbt source freshness --output target/source_freshness.json`](/reference/commands/source#configuring-source-freshness-output)|Output of "freshness" information to a different path.|
+|[`dbt source freshness --select "source:snowplow"`](/reference/commands/source#specifying-sources-to-snapshot)|Checks the "freshness" for specific sources.|
```

Comment:
i think the snowplow here is just the source name. would it be better to have a placeholder so users don't think they have to write snowplow? e.g. `dbt source freshness --select "source:source_name"`